src/main/java/org/jenkinsci/plugins/sonargerrit/gerrit/ReviewConfig.java

```diff
@@ -39,15 +39,19 @@ public class ReviewConfig extends AbstractDescribableImpl<ReviewConfig> {
    * */
   @Nonnull private String issueCommentTemplate = DescriptorImpl.ISSUE_COMMENT_TEMPLATE;
 
+  private boolean omitDuplicateComments = DescriptorImpl.ISSUE_OMIT_DUPLICATE_COMMENTS;
+
   public ReviewConfig(
       IssueFilterConfig issueFilterConfig,
       String noIssuesTitleTemplate,
       String someIssuesTitleTemplate,
-      String issueCommentTemplate) {
+      String issueCommentTemplate,
+      boolean omitDuplicateComments) {
```

Comment:
Not needed. Also this constructor signature should not change since it could be used by a third party plugin.

```suggestion
      String issueCommentTemplate) {
```